src/common/index.ts

```diff
@@ -34,6 +29,16 @@ export class RedditPostData {
   @property author: string;
 }
 
+export class RedditPost {
+
+  constructor(json: any) {
```

Comment:
Lets call these `json` variables (everywhere) `data` or something domain-specific. With `json` one could think this is a JSON string.